plugins/inventory/serf.py

```diff
@@ -37,9 +45,22 @@
 _key = 'serf'
 
 
+def _serf_client():
+    kwargs = {}
+
+    rpc_addr = os.getenv('SERF_RPC_ADDR')
+    if rpc_addr:
+        kwargs['host'], kwargs['port'] = rpc_addr.split(':')
+
+    rpc_auth = os.getenv('SERF_RPC_AUTH')
+    if rpc_auth:
+        kwargs['rpc_auth'] = rpc_auth
+
+    return SerfClient(**kwargs)
```

Comment:
Once https://github.com/KushalP/serfclient-py/pull/17 is cleaned up and merged, you can simplify this method to be:

```python
def _serf_client():
    environment_config = EnvironmentConfig()
    return SerfClient(host=env.host, port=env.port, rpc_auth=env.auth_key)
```